tools/fpgadiag/mux.py

```diff
@@ -140,10 +141,17 @@ def validate(self, opts):
             return False
         if int(opts["begin"]) % int(opts["multi-cl"]) > 0:
             return False
+        if int(opts["end"]) % int(opts["multi-cl"]) > 0:
```

Comment:
You don't need to check "end", because the actual end value is computed using "begin" and "multi-cl". This is what Nalini once told me. You can double check with her.

Reply:
This is just filtering out invalid configuration combinations. I think it shouldn't hurt to keep this here.